ansible/roles/host-ocp4-provisioner/templates/clouds.yaml.j2

```diff
@@ -6,8 +6,8 @@ clouds:
       username: "{{ guid }}-user"
       password: "{{ hostvars['localhost']['heat_user_password'] }}"
 {% else %}
-      username: "{{ osp_auth_username }}"
-      password: "{{ osp_auth_password }}"
+      username: "{{ osp_auth_username_member | d(osp_auth_username) }}"
```

Comment:
I just don't think we need this. `osp_auth_username|password_member` is always going to be set now to either a user defined value...or [what is defined here](https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-osp-project-create/tasks/main.yaml#L12-L18). We can get rid of this whole conditional and just replace the currently hardcoded `username|password` with the `osp_auth_username|password_member` vars.

I don't even think we should default back to `osp_auth_username|password` any longer. Those should _only_ be used for initially creating the project, user, etc. and then not again until we delete everything.

Reply:
I agree. My understanding of this role is that it should always be called once the cloud resources have been deployed, hence a user with `_member_` role exists.